ceph-releases/kraken/ubuntu/16.04/daemon/osd_scenarios/osd_common.sh

```diff
@@ -14,7 +14,7 @@ start_osd() {
   else
     OSD_WEIGHT=$(df -P -k "$OSD_PATH" | tail -1 | awk '{ d= $2/1073741824 ; r = sprintf("%.2f", d); print r }')
   fi
-  ceph "${CLI_OPTS[@]}" --name=osd."${OSD_ID}" --keyring="$OSD_KEYRING" osd crush create-or-move -- "${OSD_ID}" "${OSD_WEIGHT}" "${CRUSH_LOCATION}"
+  ceph "${CLI_OPTS[@]}" --name=osd."${OSD_ID}" --keyring="$OSD_KEYRING" osd crush create-or-move -- "${OSD_ID}" "${OSD_WEIGHT}" "${CRUSH_LOCATION_DEFAULT[@]}"
```

Comment:
Should still be CRUSH_LOCATION, not CRUSH_LOCATION_DEFAULT, right?

Same for other files.